data/wsl/create_unattend_raw.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```
IMHO this script should terminate on errors.